### ISSUE TITLE
Use new docker-compose option to run as a specified user

### ISSUE THREAD

**Issue by IanVS** [artificialio/sane#121]
Once https://github.com/IanVS/fig/commit/86b723e2273e9bac1a6f5b3299ee1394bfa3ec9c lands in a release of docker-compose, we can refactor new.js to run docker-compose as the current user, rather than root.  The user option can then be removed from the docker-compose.yml template.

**Comment by Globegitter** [comment]
@IanVS I wonder if @kriswill new docker image https://github.com/artificialio/docker-sails/tree/master/nvm-iojs-latest would help with that as well. Any idea @kriswill ?

**Comment by IanVS** [comment]
Yeah, if iojs isn't installed as root (like node was in the other image being used) that would help a lot.
